Dockerfile

```diff
@@ -141,12 +141,6 @@ COPY apps apps
 COPY internal internal
 
 
-FROM centos8-build-golang-base AS centos8-build-diagnostics
```

Comment:
This PR has probably some combined commits from the other `Removed diagnostics service` PR that may not be intended to be here.

Reply:
Fixed!